bigquery_etl/docs/generate_docs.py

```diff
@@ -0,0 +1,147 @@
+"""Generates documentation for a project."""
+
+from argparse import ArgumentParser
+import os
+from pathlib import Path
+import re
+import shutil
+import yaml
+
+from bigquery_etl.util import standard_args
+
+DEFAULT_PROJECTS = ["mozfun/"]
+DOCS_FILE = "README.md"
+DOCS_DIR = "docs/"
+INDEX_MD = "index.md"
+SQL_REF_RE = r"@sql\((.+)\)"
+
+parser = ArgumentParser(description=__doc__)
+parser.add_argument(
+    "--project_dirs",
+    "--project-dirs",
+    help="Directories of projects documentation is generated for.",
+    nargs="+",
+    default=DEFAULT_PROJECTS,
+)
+parser.add_argument(
+    "--docs_dir",
+    "--docs-dir",
+    default=DOCS_DIR,
+    help="Directory containing static documentation.",
+)
```

Comment:
My thinking here: We might have some static documentation in `docs/` and want to merge it with the generated documentation.